base_report_to_printer/views/printing_server.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record model="ir.ui.view" id="printing_server_view_form">
+        <field name="name">printing.server.form (in base_report_to_printer)</field>
+        <field name="model">printing.server</field>
+        <field name="arch" type="xml">
+            <form string="Servers">
```

Comment:
header string is not used anymore

Reply:
I removed the strings from the form tags.